tests/test_core.py

```diff
@@ -3968,6 +3968,43 @@ def test_dlfcn_feature_in_lib(self):
       '''
     self.do_run(src, 'float: 42.\n')
 
+  @needs_dylink
+  @no_wasm64('TODO: asyncify for wasm64')
+  def test_dlfcn_asyncify(self):
+    self.set_setting('ASYNCIFY')
+    self.set_setting('EXIT_RUNTIME', 1)
```

Comment:
Is this needed?

Reply:
Sorry, I meant is `EXIT_RUNTIME` needed?

Reply:
i.e. Can you remove `EXIT_RUNTIME` here and still have the test pass?

Reply:
If not perhaps add a comment about why its needed?

Reply:
It can be removed for now as the test case passes without `EXIT_RUNTIME`. (The test case had failed without `EXIT_RUNTIME` on the commit <https://github.com/emscripten-core/emscripten/pull/15893/commits/92148dcd3b09a96362555f25d6981a6c4e7262ba>, as far as I can remember.)